NangPaGo-be/src/main/java/com/mars/NangPaGo/domain/ingredient/controller/IngredientElasticController.java

```diff
@@ -0,0 +1,30 @@
+package com.mars.NangPaGo.domain.ingredient.controller;
+
+import com.mars.NangPaGo.domain.ingredient.service.IngredientElasticSynchronizer;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RequiredArgsConstructor
+@RequestMapping("api/ingredient")
+@RestController
+public class IngredientElasticController {
+    @Autowired private IngredientElasticSynchronizer ingredientElasticSynchronizer;
```

Comment:
클래스에 `@RequiredArgsConstructor`를 붙였으니, `@Autowired`는 빼도 되겠네요.
```suggestion
    private final IngredientElasticSynchronizer ingredientElasticSynchronizer;
```

Reply:
ae8873357ed41002ff351865169e9642726a8bf6 👌🏻 반영 완료했습니다~